Lib/test/librptest/ours.py

```diff
@@ -0,0 +1,32 @@
+
+import types
+
+from .core import mark_test_case_or_reason, get_marked_test_cases, add_subst, get_substitutions as core_get_subs
+
+
+'''
+Front End for own test cases in imported test files.
+'''
```

Comment:
Docstrings have to be the first statement of a file for it to register; could you move this to the top?